theme.js

```diff
@@ -978,6 +978,13 @@ export default () => ({
       },
     },
 
+  '.inline': {
+    container: {
+      width: window.width,
+        position: 'relative',
```

Comment:
Wow so sorry for all this time! 

So I tested with both ListView and ScrollView components.

Check out the gifs below.. 
OBS: Take in consideration that I just changed NavigationBar and that there is a `NavigationHeader` this why the extra space on top.

**Not working - not using inline `styleName`**

![navigationbar-](https://cloud.githubusercontent.com/assets/4806269/21558996/86e15bd8-ce2a-11e6-8ed1-6d8b9403d24f.gif)

**Working - using inline `styleName` + ListView**

![navigationbar-working](https://cloud.githubusercontent.com/assets/4806269/21559001/a7d85238-ce2a-11e6-9815-e68c81fee9ad.gif)

**Working - not using inline `styleName` + ScrollView**

![navigationbar-working-scrollview](https://cloud.githubusercontent.com/assets/4806269/21559004/bdd0c246-ce2a-11e6-81f8-ea6f11eec5ff.gif)